.github/workflows/integration.yml

```diff
@@ -467,4 +513,4 @@ jobs:
         # allowed-failures: docs, linters
         # allow all jobs to be skipped in case of a PR run
         allowed-skips: acceptance, phpunit, win-checkout, code-ql, acceptance-update, blue-green-66-67
-        jobs: ${{ toJSON(needs) }}
```

Comment:
you need to fix line 494

`run: php .gitlab/bin/blue-green-check.php # TODO: Move script to .github after migration`

Reply:
I'll restore the .gitlab folder and delete it in a different PR, there are more scripts that are missing.